src/PrestaShopBundle/Twig/DataFormatterExtension.php

```diff
@@ -96,6 +98,39 @@ public function unsetElement($array, $key)
         return $array;
     }
 
+    /**
+     * Extract a subset of an array and returns only the wanted keys.
+     * If $extractedKeys is an associative array you can even rename the
+     * keys of the extracted array.
+     *
+     * ex:
+     *  extractArray(['first_name' => 'John', 'last_name' => 'Doe'], ['first_name']) => ['first_name' => 'John']
+     *  extractArray(['first_name' => 'John', 'last_name' => 'Doe'], ['first_name' => 'name']) => ['name' => 'John']
+     *
+     * @param array $array
+     * @param array $extractedKeys
+     *
+     * @return array
+     */
+    public function extractArray(array $array, array $extractedKeys)
```

Comment:
This reminds me of the Laravel Helper `array_pluck`: https://laravel.com/docs/5.7/helpers#method-array-pluck

Reply:
you're right, although the format is different and it only extract one field
do you want me to change the name? I wasn't 100% sure about the naming but I could not find any better one